sdks/java/io/kafka/pom.xml

```diff
@@ -40,6 +40,35 @@
             <skip>true</skip>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-shade-plugin</artifactId>
+          <executions>
+            <execution>
+              <phase>package</phase>
+              <goals>
+                <goal>shade</goal>
+              </goals>
+              <configuration>
+                <artifactSet>
+                  <includes>
+                    <include>com.google.guava:guava</include>
+                  </includes>
+                </artifactSet>
+                <relocations>
+                  <relocation>
+                    <pattern>com.google.common</pattern>
+                    <shadedPattern>org.apache.beam.spark.relocated.com.google.common</shadedPattern>
+                  </relocation>
```

Comment:
@dhalperi Yup, copy paste mistake.

@davorbonaci I'll change in spark runner pom as well then